mu4/notation/CMakeLists.txt

```diff
@@ -79,6 +85,24 @@ set(MODULE_SRC
     ${CMAKE_CURRENT_LIST_DIR}/internal/msczmetareader.h
     ${CMAKE_CURRENT_LIST_DIR}/internal/notationplayback.cpp
     ${CMAKE_CURRENT_LIST_DIR}/internal/notationplayback.h
+    ${CMAKE_CURRENT_LIST_DIR}/view/notationpaintview.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/view/notationpaintview.h
+    ${CMAKE_CURRENT_LIST_DIR}/view/notationviewinputcontroller.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/view/notationviewinputcontroller.h
```

Comment:
Can this class be placed in internal?

Reply:
it is logical to view related, so it is better that he was in view